e2e/app/ProfilePage.ts

```diff
@@ -0,0 +1,58 @@
+import {Page} from 'puppeteer';
+import {waitUntilFindTexts} from '../driver/waitFuncs';
+import TextBox from './aou-elements/TextBox';
+import AuthenticatedPage from './page-mixin/AuthenticatedPage';
+import {SideNav} from './page-mixin/SideNav';
+
+
+export const FIELD_LABEL = {
+  TITLE: 'Profile',
+  FIRST_NAME: 'First Name',
+  LAST_NAME: 'Last Name',
+  CONTACT_EMAIL: 'Contact Email',
+  CURRENT_POSITION: 'Your Current Position',
+  ORGANIZATION: 'Your Organization',
+  CURRENT_RESEARCH_WORK: 'Current Research Work',
+  ABOUT_YOU: 'About You',
+  INSTITUTION: 'Institution',
+  ROLE: 'Role',
+  DISCARD_CHANGES: 'Discard Changes',
+  SAVE_PROFILE: 'Save Profile',
+};
+
+export default class Profile extends AuthenticatedPage {
+
+  public page: Page;
```

Comment:
Another unnecessary reference to Page object – remove in favor of this.puppeteerPage.

Reply:
side-effect of mixin. it is all being removed.